src/NServiceBus.Msmq.AcceptanceTests/When_Audit_OverrideTimeToBeReceived_set_and_native_receive_tx.cs

```diff
@@ -12,12 +11,17 @@ public class When_Audit_OverrideTimeToBeReceived_set_and_native_receive_tx : NSe
         [Test]
         public async Task Endpoint_should_not_start_and_show_error()
         {
-            var context = await Scenario.Define<Context>()
-                .WithEndpoint<Endpoint>()
-                .Done(c => c.EndpointsStarted)
-                .Run();
-
-            Assert.True(context.Exceptions.First().Message.Contains("Setting a custom OverrideTimeToBeReceived for audits is not supported on transactional MSMQ."));
+            try
```

Comment:
Better to use Assert.That(async () => , Throws....)

On Thursday, March 17, 2016, Tim Bussmann <notifications@github.com> wrote:

> In
> src/NServiceBus.Msmq.AcceptanceTests/When_Audit_OverrideTimeToBeReceived_set_and_native_receive_tx.cs
> <https://github.com/Particular/NServiceBus/pull/3603#discussion_r56492601>
> :
>
> > @@ -12,12 +11,17 @@ public class When_Audit_OverrideTimeToBeReceived_set_and_native_receive_tx : NSe
> >          [Test]
> >          public async Task Endpoint_should_not_start_and_show_error()
> >          {
> > -            var context = await Scenario.Define<Context>()
> > -                .WithEndpoint<Endpoint>()
> > -                .Done(c => c.EndpointsStarted)
> > -                .Run();
> > -
> > -            Assert.True(context.Exceptions.First().Message.Contains("Setting a custom OverrideTimeToBeReceived for audits is not supported on transactional MSMQ."));
> > +            try
>
> I think you can use Assert.Throws(async () => await...)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3603/files/c5aab2585f454a5164a2045b235748b680c90eef#r56492601>
>